pkg/cmd/purge/purge.go

```diff
@@ -0,0 +1,100 @@
+package purge
+
+import (
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	msg "github.com/aziontech/azion-cli/messages/purge"
+
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/spf13/cobra"
+)
+
+var (
+	urls      string
+	wildcard  string
+	cachekeys string
+	Layer     string
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           msg.Usage,
```

Comment:
Wouldn't we adopt message constants?

Reply:
@gabriel-azion can you update this along with the messages?